tests/Clarifai/Request/RequestHandlerTest.php

```diff
@@ -136,74 +136,7 @@ public function testRequestException()
             ->end();
         $this->http->setUp();
 
-        $clarifai = new RequestHandler('', '');
+        $clarifai = new RequestHandler('');
         $clarifai->handleRequest('GET', 'http://localhost:8082/foo', []);
     }
-
-    public function testRequestWithToken()
-    {
-        $clientId = 'client_id';
-        $clientSecret = 'client_secret';
-        $method = 'method';
-        $path = 'path';
-        $token = 'token';
-        $tokenType = 'token_type';
-        $result = 'result';
-
-        // Creates a partially mock of RequestHandler with mocked `handleRequest` method
-        $clarifai = \Mockery::mock(
-            'DarrynTen\Clarifai\Request\RequestHandler[handleRequest]',
-            [
-                $clientId,
-                $clientSecret,
-            ]
-        );
-
-        $clarifai->shouldReceive('handleRequest')
-            ->once()
-            ->with(
-                'POST',
-                'https://api.clarifai.com/v1/token',
-                [
-                    'form_params' => [
-                        'grant_type' => 'client_credentials',
-                        'client_id' => $clientId,
-                        'client_secret' => $clientSecret,
-                    ],
-                ]
-            )
-            ->andReturn(
-                (array)[
-                    'expires_in' => '60',
-                    'access_token' => $token,
-                    'token_type' => $tokenType,
-                ]
-            );
-
-        $clarifai->shouldReceive('handleRequest')
-            ->twice()
-            ->with(
-                $method,
-                'https://api.clarifai.com/v2/'.$path,
-                [
-                    'headers' => [
-                        'Authorization' => $tokenType . ' ' . $token,
-                        'User-Agent' => 'Clarifai PHP (https://github.com/darrynten/clarifai-php);v0.9.3;' . phpversion(),
-                    ],
-                ],
-                []
-            )
-            ->andReturn($result);
-
-        // Only one token request should be made to Clarifai API
-        $this->assertEquals(
-            $result,
-            $clarifai->request($method, $path)
-        );
-
-        $this->assertEquals(
-            $result,
-            $clarifai->request($method, $path)
-        );
-    }
```

Comment:
As far as I can tell there are no endpoints for just testing your API key with v2.